apps/grader-host/src/lib/config.js

```diff
@@ -26,7 +26,7 @@ const ConfigSchema = z.object({
   jobLogGroup: z.string().default('grading-jobs-debug'),
   reportLoad: z.boolean().default(false),
   reportIntervalSec: z.number().default(10),
-  graderDockerMemory: z.number().default(1 << 30), // 1GiB
+  graderDockerMemory: z.number().default(2 << 30), // 2GiB
```

Comment:
This unfortunately does not do what you want it to:

```sh
$ node -e "console.log(2 << 30)"
-2147483648
```

Reply:
ha, that's embarrassing. Should be fixed now.